launchable/test_runners/bazel.py

```diff
@@ -56,6 +58,35 @@ def f(case: TestCase, suite: TestSuite, report_file: str) -> TestPath:
 
     client.path_builder = f
     client.check_timestamp = False
-    client.scan(base, '**/test.xml')
+
+    if build_event_json:
+        lists = parse_build_event_json(build_event_json)
+        for l in lists:
+            path = Path(base).joinpath(l).resolve()
+            client.scan(path, "**/test.xml")
+    else:
+        client.scan(base, '**/test.xml')
 
     client.run()
+
+
+def parse_build_event_json(path: str) -> List[str]:
+    build_list: List[str] = []
+    try:
+        f = open(path)
+    except Exception as e:
+        click.echo(click.style(
+            "Can not open build event json file {}".format(path), "yellow"), err=True)
+        return build_list
+
+    for line in f:
+        d = json.loads(line)
+        if "id" in d:
```

Comment:
there is a full path of the report file in a testResult field.
Although I wonder that should I use it, I decided to use the label field.
because it is difficult to prepare test data

```json
{
  "id": {
    "testResult": {
      "attempt": 1,
      "configuration": {
        "id": "86e57527e1c8bb10edc853e734cb858f8159d8f3e0a4df9ceb16f80aad784b93"
      },
      "label": "//src/test/java/com/ninjinkun:mylib_test6",
      "run": 1,
      "shard": 1
    }
  },
  "testResult": {
    "executionInfo": {
      "strategy": "darwin-sandbox"
    },
    "status": "PASSED",
    "testActionOutput": [
      {
        "name": "test.log",
        "uri": "file:///private/var/tmp/_bazel_yabuki-ryosuke/3f9f29672c16363f9fe4cad29f21ed46/execroot/__main__/bazel-out/darwin-fastbuild/testlogs/src/test/java/com/ninjinkun/mylib_test6/test.log"
      },
      {
        "name": "test.xml",
        "uri": "file:///private/var/tmp/_bazel_yabuki-ryosuke/3f9f29672c16363f9fe4cad29f21ed46/execroot/__main__/bazel-out/darwin-fastbuild/testlogs/src/test/java/com/ninjinkun/mylib_test6/test.xml"
      }
    ],
    "testAttemptDurationMillis": "2577",
    "testAttemptStartMillisEpoch": "1621924928775"
  }
}
```
```